apypie/api.py

```diff
@@ -36,19 +36,22 @@ class Api:
     def __init__(self, **kwargs):
         self.uri = kwargs.get('uri')
         self.api_version = kwargs.get('api_version', 1)
-        self.language = kwargs.get('language')
```

Comment:
please don't remove this, I'll need it soon ;)

Reply:
setting it on the session isn't good enough?

Reply:
nope. I want it for the generation of the cache filename for specific langs :)